docs/tilesource_options.rst

```diff
@@ -59,15 +59,15 @@ A band definition is an object which can contain the following keys:
 
 - ``nodata``: the value to use for missing data.  null or unset to not use a nodata value.
 
-- ``composite``: either 'lighten' or 'multiply'.  Defaults to 'lighten' for all except the alpha band.
+- ``composite``: either 'lighten' or 'multiply'.  Defaults to 'lighten' for all except the alpha band. Read more about blend modes and see examples `here <blend_modes_>`_.
 
 - ``clamp``: either True to clamp (also called clip or crop) values outside of the [min, max] to the ends of the palette or False to make outside values transparent.
 
 - ``dtype``: if specified, cast the intermediate results to this data type.  Only the first such value is used, and this can be specified as a base key if ``bands`` is specified.  Normally, if a style is applied, the intermediate data is a numpy float array with values from [0,255].  If this is ``uint16``, the results are multiplied by 65535 / 255 and cast to that dtype.  If ``float``, the results are divided by 255.  If ``source``, this uses the dtype of the source image.
 
 - ``axis``: if specified, keep on the specified axis (channel) of the intermediate numpy array.  This is typically between 0 and 3 for the red, green, blue, and alpha channels.  Only the first such value is used, and this can be specified as a base key if ``bands`` is specified.
 
-- ``icc``: by default, sources that expose ICC color profiles will apply those profiles to the image data, converting the results to the sRGB profile.  To use the raw image data without ICC profile adjustments, specify an ``icc`` value of ``false``.  If the entire style is ``{"icc": false}``, the results will be the same as the default bands with only the adjustment being skipped.  Similarly, if the entire style is ``{"icc": true}``, this is the same as the default style with where the adjustment is applied.  Besides a boolean, this may also be a string with one of the intents defined by the PIL.ImageCms.Intents enum.  ``true`` is the same as ``perceptual``.   Note that not all tile sources expose ICC color profile information, even if the base file format contains it.
+- ``icc``: by default, sources that expose ICC color profiles (PIL, OpenJPEG, TIFF, TiffFile) will apply those profiles to the image data, converting the results to the sRGB profile.  To use the raw image data without ICC profile adjustments, specify an ``icc`` value of ``false``.  If the entire style is ``{"icc": false}``, the results will be the same as the default bands with only the adjustment being skipped.  Similarly, if the entire style is ``{"icc": true}``, this is the same as the default style with where the adjustment is applied.  Besides a boolean, this may also be a string with one of the intents defined by the PIL.ImageCms.Intents enum.  ``true`` is the same as ``perceptual``.   Note that not all tile sources expose ICC color profile information, even if the base file format contains it.
```

Comment:
This source list isn't complete (also OpenSlide and OMETiff), plus there could be other sources not in our monorepo (e.g., iSyntax) that expose this, too.  I fear that listing these explicitly will become out of date at some point without us noticing.

Reply:
I'll add OpenSlide and OMETiff for now, but I do think there needs to be a list somewhere. I had searched the code to find mentions of "icc", and the fact that I missed some demonstrates that it wouldn't be easy for a user to discover this list on their own. The `OMETiffTileSource` code doesn't mention "icc" at all, but I now see that it inherits that functionality from `TiffFileTileSource`.